stress/common/src/main/java/alluxio/stress/common/GeneralBenchSummary.java

```diff
@@ -36,18 +33,4 @@ public float getThroughput() {
   public void setThroughput(float throughput) {
     mThroughput = throughput;
   }
-
-  /**
-   * @return the errors
-   */
-  public Map<String, List<String>> getErrors() {
-    return mErrors;
-  }
-
-  /**
-   * @param errors the errors
-   */
-  public void setErrors(Map<String, List<String>> errors) {
-    mErrors = errors;
-  }
```

Comment:
Now Summary will store the result information about every nodes in a map, so the Error information are now involved in that place, which is mNodeResults in the [MultipleNodeBenchSummary]